commands/createfixpullrequests.go

```diff
@@ -157,64 +157,85 @@ func (cfp *CreateFixPullRequestsCmd) fixIssuesSeparatePRs(fixVersionsMap map[str
 	return
 }
 
-func (cfp *CreateFixPullRequestsCmd) fixIssuesSinglePR(fixVersionsMap map[string]*utils.FixVersionInfo) (err error) {
+func (cfp *CreateFixPullRequestsCmd) fixIssuesSinglePR(fixVersionsMap map[string]*utils.FixDetails) (err error) {
+	leastOneFix := false
 	log.Info("-----------------------------------------------------------------")
 	log.Info("Start aggregated packages fix")
 	aggregatedFixBranchName, err := cfp.gitManager.GenerateAggregatedFixBranchName(fixVersionsMap)
 	if err != nil {
 		return
 	}
-	if err = cfp.createFixBranch(aggregatedFixBranchName, false); err != nil {
+	log.Info("Creating branch", aggregatedFixBranchName, "...")
+	err = cfp.gitManager.CreateBranchAndCheckout(aggregatedFixBranchName)
+	if err != nil {
 		return
 	}
 	// Fix all packages in the same branch
 	for impactedPackage, fixVersionInfo := range fixVersionsMap {
-		if err = cfp.updatePackageToFixedVersion(impactedPackage, fixVersionInfo); err != nil {
-			log.Error("Could not fix impacted package", impactedPackage, "as part of the PR. Skipping it. Cause:", err.Error())
+		shouldFix, err := cfp.updatePackageToFixedVersion(fixVersionInfo)
+		leastOneFix = leastOneFix || shouldFix
+		if err != nil {
+			log.Debug("Could not fix impacted package", impactedPackage, "as part of the PR. Skipping it. Cause:", err.Error())
```

Comment:
log.Debug("Could not fix impacted package", impactedPackage, "as part of the PR. Skipping it. Cause:", err.Error())
-->
log.Info("Skipped fixing impacted package", impactedPackage, "as part of the PR. Reason:", err.Error())